src/config_models/cli_args.rs

```diff
@@ -170,6 +170,21 @@ pub struct Args {
     #[clap(long)]
     pub(crate) max_mempool_num_tx: Option<usize>,
 
+    /// Restrict the number of inputs for transcations that this node deals
+    /// with.
+    ///
+    /// If this value is set, the node will not create transactions with more
+    /// inputs than this limit. It will also not accept such transactions from
+    /// peers into its mempool. And if the node is either upgrading transaction
+    /// proofs on the network or composing, it will never merge two transactions
+    /// that would exceed this limit. Does not affect block validity, i.e.
+    /// consensus. So a block with more than this number of inputs may be
+    /// considered value. Also does not affect which block proposals that a
+    /// guessing node will attempt to solve the PoW puzzle for. Set to "none" to
+    /// disable this limit.
+    #[clap(long, value_parser = parse_optional_usize, default_value = "28")]
```

Comment:
It is not obvious to me that the default string will parse correctly. Please add it to the test case below.